### PR TITLE
[1LP][RFR] pdf download popup fix

### PR DESCRIPTION
**Pull request by mmojzis**
This PR is fixing change in 5.10 where pdf instead of being downloaded is displayed as popup tab.
Also setting importance from high to medium.

@lina-nikiforova Maybe I should reassign these test cases to you?

### CODE REVIEW

File: cfme/networks/views.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ class NetworkProviderToolBar(View):
 class NetworkProviderDetailsToolBar(NetworkProviderToolBar):
     """ Represents provider details toolbar """
     monitoring = Dropdown(text='Monitoring')
-    download = Button(title='Download summary in PDF format')
+    download = Button(title='Print or export summary')
```
Comment (mmojzis) [inline cfme/networks/views.py]: @quarckster Hello Dima, please can you tell me how to use version.picker here?
I need to keep the old text for 5.9 backwards compatibility.
Thanks

Comment (quarckster) [reply cfme/networks/views.py]:
```python
from cfme.utils.version import VersionPicker, Version
...
download = VersionPicker(
  Version.LOWEST: Button(title='Print or export summary'),
  "5.10": Button(title='New title')
)
```

File: cfme/tests/networks/test_sdn_downloads.py [hunk 2]
```diff
@@ -33,13 +33,25 @@ def download(appliance, objecttype, extension):
     view = navigate_to(objecttype, 'All')
     if appliance.version >= '5.10' and extension == 'pdf':
         view.toolbar.download.item_select("Print or export as PDF")
+        handle_extra_tabs(view)
     else:
         view.toolbar.download.item_select("Download as {}".format(extensions_mapping[extension]))
 
 
-def download_summary(spec_object):
+def download_summary(spec_object, appliance):
```
[the remark this answers is not in the document]
Comment (quarckster) [reply cfme/tests/networks/test_sdn_downloads.py]: And from where `VersionPicker` will take `appliance.version`?

File: cfme/tests/networks/test_sdn_downloads.py [hunk 3]
```diff
@@ -56,6 +68,8 @@ def test_download_lists_base(filetype, collection_type, appliance):
     Polarion:
         assignee: mmojzis
         initialEstimate: 1/10h
+        casecomponent: web_ui
```
Comment (digitronik) [inline cfme/tests/networks/test_sdn_downloads.py]: Required: `WebUI` please.

File: cfme/tests/networks/test_sdn_downloads.py [hunk 4]
```diff
@@ -74,12 +88,14 @@ def test_download_pdf_summary(appliance, collection_type, provider):
...
```
Comment (digitronik) [inline cfme/tests/networks/test_sdn_downloads.py]: as above

**Pull request merged**